packages/react/src/authentication/hooks/types/TokenKinds.types.ts

```diff
@@ -0,0 +1,7 @@
+enum TokenKinds {
```

Comment:
Do not redefine types that are defined in other packages. This type can be removed and UserToken type that is defined in @farfetch/blackout-client should be used instead

Reply:
and done 😀